MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult.swift

```diff
@@ -27,13 +27,13 @@ import Foundation
 /// Class that groups account and token information.
 @objc public class MSALNativeAuthUserAccountResult: NSObject {
     /// The account object that holds account information.
-    @objc public let account: MSALAccount
+    @objc public var account: MSALAccount
 
-    let authTokens: MSALNativeAuthTokens
+    var authTokens: MSALNativeAuthTokens
     let configuration: MSALNativeAuthConfiguration
     private let cacheAccessor: MSALNativeAuthCacheInterface
 
-    /// Get the ID token for the account.
+    /// Get the latest ID token for the account.
     @objc public var idToken: String? {
         authTokens.rawIdToken
```

Comment:
@spetrescu84 did you consider retrieving the id token from the cache here? Maybe that would be a cleaner way, as you wouldn't have to make the `account` property mutable and would avoid most of the new changes.

I'm not sure if the performance would be affected, though.